test/drawers/drawerTests.ts

```diff
@@ -22,17 +21,17 @@ class MockAnimator implements Plottable.Animator {
 }
 
 function createMockDrawer(dataset: Plottable.Dataset) {
-  var drawer = new Plottable.Drawer(dataset);
+  const drawer = new Plottable.Drawer(dataset);
```

Comment:
We should be thinking of the scope of this variable.

The scope of this variable is only to the inside of the function, and thus the intention of this variable is to hold specifically a reference of a mock drawer that is created and returned.